fetch.bs

```diff
@@ -2172,6 +2236,132 @@ exploits pertain to those <a for=request>types</a>. Also, considering "<code>ima
 compatible with deployed content.
 
 
+<h3 id=token-binding>Token Binding</h3>
+<p>In order to protect security tokens like <a>credentials</a> and OAuth tokens, user agents
+and servers can use Token Binding to cryptographically associate a given
+security token with the user agent, thereby making it more difficult for
+attackers to abuse the capabilities the security token grants.
+This method of associating credentials with a user agent is referred to as
+<dfn export id=concept-token-binding>token binding</dfn>.
+
+<p>The technique is described in
+detail in [[TOKBIND-NEGOTIATION]], [[TOKBIND-PROTOCOL]], and [[TOKBIND-HTTPS]].
+
+<p>A <dfn export id=token-binding-id>token binding ID</dfn> is the representation
+of a <a for=/>token binding</a>, as described in
+<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol#section-3.2">section 3.2</a>
+of [[TOKBIND-PROTOCOL]].
+
+<p>At a very high level, a user agent negotiates the use of Token Binding when it
+sets up a TLS connection to a server.
+The user agent maintains a Token Binding key store, where it saves different
+<a for=/>token binding ID</a>s to be used with different servers.
```

Comment:
s/servers/origins/ throughout? Or is it the case that a set of coalesced connections use the same binding?